### PR TITLE
trillian: Add version 6.4.0.5

### PR DESCRIPTION
**Pull request by stasjok**
Trillian
https://www.trillian.im

### CODE REVIEW

File: bucket/trillian.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        "$file = 'trillian.ini'",
+        "if (!(Test-Path \"$persist_dir\\$file\")) {",
+        "   Write-Host 'File' $file 'does not exists. Creating.' -f Yellow",
+        "   $CONT = @(",
+        "       '[General]'",
+        "       'Use User Directory=0'",
+        "       'Global Directory=users\\global\\'",
+        "   )",
+        "   Set-Content \"$dir\\$file\" ($CONT -join \"`r`n\") -Encoding Ascii",
+        "}"
```
Comment (Ash258) [inline bucket/trillian.json:21]:
```suggestion
        "if (!(Test-Path \"$persist_dir\\trillian.ini\")) {",
        "   Set-Content \"$dir\\trillian.ini\" ('[General]', 'Use User Directory=0', 'Global Directory=users\\global\\') -Encoding Ascii",
        "}"
```

File: bucket/trillian.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            "Trillian"
+        ]
+    ],
+    "persist": [
+        "trillian.ini",
+        "users"
+    ],
+    "checkver": {
+        "url": "https://www.trillian.im/shared/scripts/news.items.js",
+        "regex": "\"windows\",\\s*([\\d.]+),\\s*([\\d]+),\\s*\"[^\"]+\",\\s+false",
```
Comment (Ash258) [inline bucket/trillian.json:35]:
```suggestion
        "regex": "\"windows\",\\s*([\\d.]+),\\s*([\\d]+),\\s*\".*?\",\\s*false",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
Blocked by lukesampson/scoop#3828

**Comment by rashil2000** [comment]
The latest URL is https://trillian.im/get/windows/6.5/trillian-v6.5.0.18.exe, but 7z cannot extract it. @issaclin32 does it work for you?

**Comment by issaclin32** [comment]
> The latest URL is https://trillian.im/get/windows/6.5/trillian-v6.5.0.18.exe, but 7z cannot extract it. @issaclin32 does it work for you?

Cannot extract either. (using 7-zip 21.07)

**Comment by stasjok** [comment]
I've updated autoupdate URL.

**Comment by rashil2000** [comment]
Have you tested the manifest by installing it?

**Comment by stasjok** [comment]
> Have you tested the manifest by installing it?

I've updated version that was already installed for me on my computer with this manifest. Looks like versions 6.4 and earlier are downloaded from ceruleanstudios.com, but since 6.5 they moved to trillian.im. So autoupdate was broken. But thanks for your message I noticed it and fixed for now.

**Pull request merged**